monkestation/code/modules/mob/living/carbon/human/species_type/ipc.dm

```diff
@@ -438,3 +438,13 @@
 		)
 
 	return to_add
+
+/datum/species/ipc/handle_chemical(datum/reagent/chem, mob/living/carbon/human/H, seconds_per_tick, times_fired)
+	if(chem?.synthetic_boozepwr)
+		var/booze_power = chem?.synthetic_boozepwr
```

Comment:
```suggestion
	var/booze_power = chem?.synthetic_boozepwr
	if(booze_power)
```
doing `?.` twice is redundant, you can just do this instead

also plz no single-letter vars (rename `H` to `ipc` or something)